yolox/models/build.py

```diff
@@ -48,44 +52,59 @@ def create_yolox_model(
         device = "cuda:0" if torch.cuda.is_available() else "cpu"
     device = torch.device(device)
 
-    assert name in _CKPT_FULL_PATH, f"user should use one of value in {_CKPT_FULL_PATH.keys()}"
-    exp: Exp = get_exp(exp_name=name)
-    exp.num_classes = num_classes
-    yolox_model = exp.get_model()
-    if pretrained and num_classes == 80:
-        weights_url = _CKPT_FULL_PATH[name]
-        ckpt = load_state_dict_from_url(weights_url, map_location="cpu")
-        if "model" in ckpt:
-            ckpt = ckpt["model"]
-        yolox_model.load_state_dict(ckpt)
+    assert name in _CKPT_FULL_PATH or name == "yolox_custom", \
+        f"user should use one of value in {_CKPT_FULL_PATH.keys()} or \"yolox_custom\""
+    if name in _CKPT_FULL_PATH:
+        exp: Exp = get_exp(exp_name=name)
+        exp.num_classes = num_classes
+        yolox_model = exp.get_model()
+        if pretrained and num_classes == 80:
```

Comment:
yolox_custom and others differs in logic like `get_ckpt`, things like `load_state_dict` should be moved outside.

Reply:
I didn't quite get what you mean by this.
At this point I'm considering a completely separate from `create_yolox_model` function, just to not overflow it with parameters. 
Also using `torch.hub.load` with keyword args (like `exp_path="exp.py", ckpt_path="ckpt.pth"`) seems wrong to me and it may be avoided with a separate function.
What do you thing about it?

Reply:
My bad, resovle this.